include/ck/tensor_operation/gpu/device/device_index_pool_bwd.hpp

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <vector>
+
+#include "ck/tensor_operation/gpu/device/device_base.hpp"
+#include "ck/utility/reduction_enums.hpp"
```

Comment:
Remove the including since it is not needed

Reply:
508d7a59d9d7b016937643044168d9d90a7dc3a5